recipes/gubbins/meta.yaml

```diff
@@ -1,18 +1,18 @@
 {% set name = "gubbins" %}
-{% set version = "2.2.1" %}
-{% set md5 = "07315be3d3c0bcaa4b94df9d21b6e63f" %}
+{% set version = "2.3.1" %}
+{% set md5 = "157e91666afc6160b4fe15327dcd375e" %}
 
 package:
   name: {{ name|lower }}
   version: {{ version }}
 
 build:
-  number: 0
+  number: 1
```

Comment:
Keep 0 for new versions.

Reply:
Travis fell over with a bioconda error, totally unrelated to this, but forcing another build fixed it.

Reply:
Andrew do you have the error? It is still failing.